staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/types_jsonschema.go

```diff
@@ -16,8 +16,26 @@ limitations under the License.
 
 package v1
 
-import (
-	"k8s.io/apimachinery/pkg/util/validation/field"
+// FieldValueErrorReason is a machine-readable value providing more detail about why a field failed the validation.
+// +enum
+type FieldValueErrorReason string
+
+const (
+	// FieldValueRequired is used to report required values that are not
+	// provided (e.g. empty strings, null values, or empty arrays).  See
+	// Required().
```

Comment:
the function references (e.g. `Required()`, `Duplicate()`, `Invalid()`, etc) don't make a lot of sense here, removed from the `field` package